components/features/VideoConference/PreviewRoom/PreviewRoom.tsx

```diff
@@ -4,9 +4,7 @@ import {
   useHMSStore,
   selectIsInPreview,
   selectLocalPeer,
-  selectIsConnectedToRoom,
-  useHMSNotifications,
-  HMSNotificationTypes,
+  selectIsConnectedToRoom
```

Comment:
Hello Alex, thanks for removing my unused imports!

Reply:
No probs rashad! I hope I don't reintroduce more in my next commits. 😀